src/components/EditAndApproveStory/EditAndApproveStory.tsx

```diff
@@ -0,0 +1,320 @@
+              label="Excerpt"
+              fullWidth
+              multiline
+              rows={2}
+              variant="outlined"
+              value={props.excerpt}
+            />
+          </Grid>
+
+          <Grid item xs={12} sm={6}>
+            <Card variant="outlined">
+              <CardMedia
+                component="img"
+                image={props.thumbnailUrl}
+                title="Test image"
+              />
+            </Card>
+          </Grid>
+
+          <Grid item xs={12} sm={6}>
+            <TextField
+              id="alt-text"
+              name="alt-text"
+              label="Alt Text"
+              fullWidth
+              InputLabelProps={{
+                shrink: true,
+              }}
+              multiline
+              rows={8}
+              variant="outlined"
+              value={props.altText}
+            />
+            <FormHelperText>
+              <a className={classes.greyLink} href="https://getpocket.com/">
+                Learn how to write alt text
+              </a>
+            </FormHelperText>
+          </Grid>
+
+          <Grid item xs={12}>
+            <TextField
+              id="thumbnail-url"
+              name="thumbnail-url"
+              label="Thumbnail URL"
+              fullWidth
+              InputLabelProps={{
+                shrink: true,
+              }}
+              variant="outlined"
+              value={props.thumbnailUrl}
+            />
+          </Grid>
+
+          <Grid item xs={12} sm={6}>
+            <TextField
+              disabled
+              id="source"
+              name="source"
+              label="Source"
+              fullWidth
+              InputLabelProps={{
+                shrink: true,
+              }}
+              variant="outlined"
+              value={props.source}
+            />
+          </Grid>
+          <Grid item xs={12} sm={6}>
+            <TextField
+              id="topic"
+              name="topic"
+              select
+              value="0"
+              label="Topic"
+              fullWidth
+              InputLabelProps={{
+                shrink: true,
+              }}
+              variant="outlined"
+            >
+              <MenuItem key="0" value="0">
+                Health
+              </MenuItem>
+              <MenuItem key="1" value="1">
+                Art
+              </MenuItem>
+              <MenuItem key="2" value="2">
+                Languages
+              </MenuItem>
+            </TextField>
+          </Grid>
+
+          <Grid item xs={12}>
+            <TextField
+              id="history"
+              name="history"
+              InputLabelProps={{
+                shrink: true,
+              }}
+              label="History"
+              fullWidth
+              multiline
+              rows={4}
+              variant="outlined"
+            />
+          </Grid>
+
+          <Grid item xs={12} className={classes.alignRight}>
```

Comment:
Let's get rid of this `Prioritize this story` toggle

Reply:
Done! Do we need custom Switch styles or should these be removed, too?

Reply:
For now, we don't need the custom switch styles

Reply:
Removed these from the theme file :+1: